content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/enforce-2fa.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Enforce Two-Factor Authentication"
+linkTitle: "Enforce 2FA"
+weight: 10
+description: >
+  Enforce 2FA for users.
+---
+
+{{% pageinfo %}}
+As an [Organization Owner](/getting-started/glossary/#organization-owner), you can enforce two-factor-authentication (2FA) for all users within your organization.
+{{% /pageinfo %}}
+
+2FA enforcement affects users belonging to your organization, including the following roles:
+
+{{% owner-member-team-member %}}
+
+{{< alert title="Note" color="primary" >}}
+If your organization uses [SAML-based single sign-on (SSO)](/getting-started/sign-in/#saml-sso), you bypass any 2FA enforcement. Your SAML provider may already require 2FA.
+{{< /alert >}}
+
+## Enforce 2FA
+
+1. Navigate to **Settings** > **Identity & Access**.
+1. Under **Two-Factor Authentication**, select **Enforce 2FA**.<br><br>
+  ![Enforce 2FA for your organization](/deepdive/Enforce2FA.png "Enforce 2FA for your organization")
+1. In the overlay that appears, select **Yes, enforce 2FA** to confirm.
+
+Users who haven't yet configured 2FA on their account get an email notification. They need to enable 2FA before signing in. For details, see steps 3–5 described in [Enable Two-Factor Authentication](/platform-deep-dive/cobalt-account/account-settings/#enable-two-factor-authentication).
+
+As an Organization Owner who enforces 2FA, you see an overlay prompting you to [enable 2FA](/platform-deep-dive/cobalt-account/account-settings/#enable-two-factor-authentication)—if you haven't already.
+
+{{< alert title="Tip" color="primary" >}}
+{{% 2fa-see-troubleshooting %}}
+{{< /alert >}}
+
+### Check the 2FA Status of Users
+
+To view the 2FA status on the user accounts within your organization, navigate to the **People** page. You see a yellow warning icon ![Warning icon](/icons/Warning.png "Warning icon") for a user when:
+
+- The user hasn't enabled 2FA—regardless of the 2FA enforcement; and
+- Your organization doesn't have SAML SSO configured.
```

Comment:
im lost on this line. Do we not show a 2FA warning icon if SAML is configured and user has 2FA enabled?

Reply:
@molfinn This info comes from the [Zendesk article](https://cobaltio.zendesk.com/hc/en-us/articles/4408918977940-Enforce-2FA-for-your-Organization). I invited a user to the test organization with SSO enabled. I can see that there is no warning icon on the People page. However, I can't complete the account setup flow and sign in with that user because I don't have access to the IdP.

Reply:
@ana-dashuk-cobalt got it - i see that now. Thank you.